src/meshapi/tasks.py

```diff
@@ -18,6 +18,21 @@
 from meshdb.settings import MESHDB_ENVIRONMENT
 
 
+@celery_app.task
+def run_uisp_on_demand_import(target_nn: int) -> None:
+    try:
+        import_and_sync_uisp_devices(get_uisp_devices(), target_nn)
+        import_and_sync_uisp_links(get_uisp_links(), target_nn)
+        sync_link_table_into_los_objects(target_nn)
+    except Exception as e:
+        logging.exception(e)
+        statsd.increment("meshdb.tasks.run_uisp_on_demand", tags=["status:failure"])
+        # TODO: (wdn) How do we alert the user if the job failed?
```

Comment:
Let's use the existing "admin notifications system". Super easy no creds required, just call this function: https://github.com/nycmeshnet/meshdb/blob/efcbdf2eef613370cac9b4045f27559a10fec3fc/src/meshapi/util/admin_notifications.py#L21

It might need a slight refactor to be a bit less model-specific but I bet that's not too bad